ex1B.py

```diff
@@ -0,0 +1,10 @@
+
+def remove_dividend_numbers(num_list: list[int], divisor: int):
+    for number in num_list[:]:  
+        if number % divisor == 0:
+            num_list.remove(number)
+    return num_list
+list1 = remove_dividend_numbers([2, 12, 100, 6, 8, 3], 2)
+print(list1)
+
```

Comment:
please add another version for the remove_dividend_numbers that uses the builtin filter function :
[https://www.programiz.com/python-programming/methods/built-in/filter](url)

also use it with a lambda function